vagrant/Vagrantfile

```diff
@@ -0,0 +1,11 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "debian/contrib-jessie64"
```

Comment:
As I suspected, old Ubuntu won't mount systemd by default:
```
==> default: /tmp/vagrant-shell: line 47: systemctl: command not found
```
What do you suggest? If you ask me I vote for Debian Stable (Jessie)